.changeset/itchy-moles-count.md

```diff
@@ -0,0 +1,9 @@
+---
+'xstate': minor
+'@xstate/fsm': minor
+'@xstate/inspect': minor
+'@xstate/react': minor
+'@xstate/vue': minor
+---
+
+add typings for interop observables (in core, affects dep versions in other packages)
```

Comment:
I don't think that other packages are impacted by this change in a meaningful way 🤔 
```suggestion
---
'xstate': patch
---

Add type for `Symbol.observable` to the `Interpreter` to improve the compatibility with RxJS.
```

Reply:
Me neither. If version bumps are not considered a minor change, then only the core package is enough.